libraries/hatch-user-management-client/src/UserManagementClientSdk.ts

```diff
@@ -0,0 +1,236 @@
+import {inject, injectable, Logger} from '@launchtray/hatch-util';
+import {WEB_SERVICES_URL} from './constants';
+import fetch from 'cross-fetch';
+import {UserManagementClient, UserServiceClientEndpoints} from './UserManagementClient';
+import {UserManagementError} from './UserManagementError';
+
+@injectable()
+export class UserManagementClientSdk implements Omit<UserManagementClient, 'getUserInfo'> {
```

Comment:
Per offline discussion, let's either implement a getUserInfo endpoint, or remove it from the interface.

Reply:
updated